packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/statement.iteration.scope

```diff
@@ -0,0 +1,33 @@
+// Start
```

Comment:
Included to make it very obvious that it's not the function, but the entire document